src/test/unit/profile-conversion.test.js

```diff
@@ -247,6 +251,116 @@ describe('converting Google Chrome profile', function () {
     checkProfileContainsUniqueTid(profile);
     expect(profile).toMatchSnapshot();
   });
+
+  it('successfully imports a chrome profile using markers of different types', async function () {
```

Comment:
I added an explicit test despite that it looks like other tests were exercizing this as well (according to the snapshot changes).